samples/bluetooth/hci_spi/prj.conf

```diff
@@ -1,10 +1,6 @@
 CONFIG_GPIO=y
 CONFIG_SPI=y
 CONFIG_SPI_SLAVE=y
-CONFIG_MAIN_STACK_SIZE=512
-CONFIG_BT=y
-CONFIG_BT_HCI_RAW=y
 CONFIG_BT_MAX_CONN=20
-CONFIG_BT_TINYCRYPT_ECC=n
 CONFIG_BT_RX_BUF_COUNT=10
 CONFIG_BT_RX_BUF_LEN=255
```

Comment:
can consider moving this to common config too

Reply:
We could do that, this one does not need 255 as far as I can see.